services/blockchain-indexer/shared/database/schema/votes.js

```diff
@@ -14,19 +14,17 @@
  *
  */
 module.exports = {
-	primaryKey: 'tempId',
+	primaryKey: ['id', 'receivedAddress'],
```

Comment:
Can we please update the implementation to contain only the aggregated values based on a combination of `sentAddress` and `receivedAddress`? Please use the `increment` and `decrement` methods when indexing or removing a block from the index respectively. We do not need the `votes_aggregates` index anymore.

```suggestion
	primaryKey: ['sentAddress', 'receivedAddress'],
```